utils/dashboard-helper.ts

```diff
@@ -28,14 +29,29 @@ export const checkLine = (line: string) => {
   return warningsFound;
 };
 
+export const getWarnings = (dashboardJson: any) => {
+  const dashLines = neatJSON(dashboardJson, {padding:2}).split('\n');
```

Comment:
I'd rather not pull in a new package (especially one that isn't at a 1.0 state yet) just to do some JSON handling. If all we're using is the ability to one-line the array values, then the following snippet might work just fine:
```js
JSON.stringify(dashboardJson, (k, v) => {
  if (Array.isArray(v)) {
    return JSON.stringify(v)
  }
  return v
}, 2);
```

This snippet is runnable and you can see that the output puts the array on a single line:
```js
console.log(JSON.stringify({id: [2], desc: 'asdfadsfa'}, (k, v) => {
  if (Array.isArray(v)) {
    return JSON.stringify(v)
  }
  return v
}, 2));
```

Reply:
Took your suggestion!